float64 motorStroke
float64 gripperStroke
bool relativeDrive
---
bool success
Feedback execution
